src/features/game/toast/ToastManager.tsx

```diff
@@ -28,8 +28,8 @@ export const ToastManager: React.FC = () => {
             "flex flex-col items-end sm:block fixed z-[99999] pointer-events-none"
           }
           style={{
-            top: `${PIXEL_SCALE * 52}px`,
-            left: `${PIXEL_SCALE * 3}px`,
+            top: `${PIXEL_SCALE * 3}px`,
```

Comment:
Was this change intended for this PR or https://github.com/sunflower-land/sunflower-land/pull/2017?

Reply:
Thanks for the catch, I have reverted the changes in this PR (I shouldn't work on 2 PRs at the same time 😂)